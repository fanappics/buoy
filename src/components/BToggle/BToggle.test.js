// import { shallow } from 'vue-test-utils'
// import BToggle from './BToggle'

describe('BToggle', () => {
  it('lets me commit', () => {
    expect(true).toBe(true)
  })
})
