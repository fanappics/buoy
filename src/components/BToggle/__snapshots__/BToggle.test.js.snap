// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BToggle renders the toggle 1`] = `
<div class="toggle-container">
    <label class="toggle">
        <input type="checkbox"> <span class="slider"><label class="text">test</label></span></label>
</div>
`;

exports[`BToggle renders the toggle disabled 1`] = `
<div class="toggle-container">
    <label class="toggle">
        <input disabled="disabled" type="checkbox"> <span class="slider"><label class="disabled text">test</label></span></label>
</div>
`;

exports[`BToggle renders the toggle prechecked 1`] = `
<div class="toggle-container">
    <label class="toggle">
        <input aria-pressed="true" type="checkbox"> <span class="slider"><label class="text">test</label></span></label>
</div>
`;
